pkg/authorization/registry/clusterrole/proxy.go

```diff
@@ -94,7 +94,7 @@ func (s *REST) Delete(ctx apirequest.Context, name string, options *metav1.Delet
 	return &metav1.Status{Status: metav1.StatusSuccess}, true, nil
```

Comment:
clarify in commit title/description which resources we're not running admission on and why